spec/controllers/harvest_schedules_controller_spec.rb

```diff
@@ -18,11 +19,23 @@
   end
 
   describe 'PUT update_all' do
+
+    before do
+      expect(schedule).to receive(:status) { 'active' }
+      expect(schedule).to receive(:update_attributes).with({ 'status' => 'stopped' })
+    end
+
     it 'update the scheduled harvets' do
       expect(HarvestSchedule).to receive(:all) { [schedule] }
-      expect(schedule).to receive(:update_attributes).with({ 'status' => 'paused' })
+      put :update_all, environment: 'staging', harvest_schedule: { status: 'stopped' }
+    end
+
+    it 'does not update individually paused scheduled harvests' do
+      expect(HarvestSchedule).to receive(:all) { [schedule, paused_schedule] }
+      expect(paused_schedule).to receive(:status) { 'paused' }
+      expect(paused_schedule).to_not receive(:update_attributes).with({ 'status' => 'stopped' })
```

Comment:
are we able to use `{status: 'stopped'}` in the `.with()`?

Reply:
done